adafruit_esp32spi/adafruit_esp32spi.py

```diff
@@ -749,6 +767,12 @@ def socket_connect(self, socket_num, dest, port, conn_mode=TCP_MODE):
             print("*** Socket connect mode", conn_mode)
 
         self.socket_open(socket_num, dest, port, conn_mode=conn_mode)
+        if conn_mode == self.UDP_MODE:
+            # UDP doesn't actually establish a connection
+            # but the socket for writing is created via start_server
+            self.start_server(port, socket_num, conn_mode)
```

Comment:
Not sure how much of an issue this is, start_server is defined underneath this fn.  It works on my pyportal, but the rest of these files seem to respect the convention of only using functions defined above.

Reply:
Can you add an example of what you're running (your `code.py`) on your device?

Reply:
I added a skeleton call flow to the description, I can clean up and put in a full code.py example.  It's specific to SSDP though, I don't want to cause confusion.